src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -102,7 +103,9 @@ const UserSubsidy = ({ children }) => {
     <>
       {/* Render the children so the rest of the page shows */}
       <UserSubsidyContext.Provider value={contextValue}>
-        {children}
+        <SubsidyRequestsContextProvider>
```

Comment:
Initially moved `SubsidyRequestsContextProvider` in [AuthenticatedUserSubsidyPage](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/src/components/app/AuthenticatedUserSubsidyPage.jsx) but this provider relies on `AppContext` and `UserSubsidyContext` so I had to move it a further bit down the tree.